src/components/analytics/Basic.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div id="expense">
+    <div class="diff-shadow">
+      <div class="flex-box">
+        <label class="pad-label ls" for="start_date">
+          <strong>Start:</strong>
+        </label>
+        <div class="s-i">
+          <input
+            name="start_date"
+            type="date"
+            v-model="startDate"
+          />
+        </div>
+        <label class="pad-label mr-l le" for="end_date">
+          <strong>End:</strong>
+        </label>
+        <div class="e-i">
+          <input
+            name="end_date"
+            type="date"
+            v-model="endDate"
+          />
+        <span v-if="dateValidation" class="form-error">{{dateValidation}}</span>
+        </div>
+        <div class="b" style="margin-left: 10px">
+          <button class="btn btn-orange" @click="fetchAnalyticsBtn">Search Analytics</button>
+        </div>
+      </div>
+      <ul class="nav nav-tabs">
+        <li class="nav-item">
+          <span :class="tab === 'order' ? 'nav-link active' : 'nav-link'">
+            <router-link @click="tab = 'order'" :to="{name: 'order-analytics'}">
+              <strong>Orders</strong>
+            </router-link>
+          </span>
+        </li>
+        <li class="nav-item">
+          <span :class="tab === 'inventory' ? 'nav-link active' : 'nav-link'">
+            <router-link @click="tab = 'inventory'" :to="{name: 'inventory-analytics'}">
+              <strong>Inventory</strong>
+            </router-link>
+          </span>
+        </li>
+        <li class="nav-item">
+          <span :class="tab === 'finance' ? 'nav-link active' : 'nav-link'">
+            <router-link @click="tab = 'finance'" :to="{name: 'finance-analytics'}">
+              <strong>Finance</strong>
+            </router-link>
+          </span>
+        </li>
+      </ul>
+      <div>
+        <router-view />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { mapActions } from 'vuex';
+import { ActionTypes } from '@/store/modules/order/actions';
+
+export default defineComponent({
+  name: 'Expense',
+  data() {
+    const path = window.location.pathname;
+    return {
+      tab: path.split('/')[2],
+      startDate: '',
```

Comment:
set some default date range (startDate and endDate). Nothing appears on the page by default.